benchmark/runtime/main.py

```diff
@@ -10,11 +10,11 @@
 from torch_geometric.datasets import Entities, Planetoid
 
 device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
-root = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data')
-Cora = Planetoid(osp.join(root, 'Cora'), 'Cora')
-CiteSeer = Planetoid(osp.join(root, 'CiteSeer'), 'CiteSeer')
-PubMed = Planetoid(osp.join(root, 'PubMed'), 'PubMed')
-MUTAG = Entities(osp.join(root, 'EntitiesMUTAG'), 'MUTAG')
+root = Path.joinpath(Path(__file__).resolve().parent.parent, 'data')
+Cora = Planetoid(str(root.joinpath('Cora')), 'Cora')
```

Comment:
```suggestion
Cora = Planetoid(str(root.joinpath('Cora')), 'Cora')
```
Why do we need to convert to `str` here? Is this needed to let the `dataset` work as expected? Shouldn't we work on supporting this on the `dataset` side first?

Reply:
There is no need to convert to `str` Dataset supports both `str` and `Path`. I have pushed a commit to fix this issue.